src/components/Accordion/_index.scss

```diff
@@ -29,6 +29,10 @@
       }
     }
 
+    &__heading {
```

Comment:
- [ ] The change looks good to me, but I feel like this only needs to be applied to our variation, while Carbon's needs to apply to theirs

Reply:
I have a feeling that it could look confuse people somehow, but then again it's just an assumption so... better to make a smaller change at first 👍